test/specs/rules/require-padding-newlines-before-line-comments.js

```diff
@@ -46,10 +46,6 @@ describe('rules/require-padding-newlines-before-line-comments', function() {
             checker.configure({ requirePaddingNewLinesBeforeLineComments: true });
         });
 
-        it('should report missing padding before line comment', function() {
```

Comment:
I'm not sure why this was removed. This seems like exact test that this rule is supposed to fail.